Users/rajeev/dd/integrations-extras/fiddler/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
```

Comment:
Could you remove this file? It was generated by the `ddev command` because of a bug, but it's now fixed

Reply:
@FlorentClarret are you talking about removing the pyproject.toml file alltogether? That will break the build, right? Can you tell me explicitly which file you are asking me to remove?

Reply:
Sure :) if you take a closer look, you will see that this `pyproject.toml` file is not the one at the root of your integration, but it is stored in this folder `Users/rajeev/dd/integrations-extras/fiddler/pyproject.toml` and we do not need this one. Remove this one, not the one at the root

Reply:
@FlorentClarret - when I remove the pyproject.toml file from my fiddler folder and run ddev, this is what I get.

(base) rajeev@Rajeevs-MacBook-Pro fiddler % ddev test fiddler
Running tests for fiddler
ERROR: missing /Users/rajeev/junk/integrations-extras-Jul-29-2022/integrations-extras/fiddler/pyproject.toml

Failed!